src/shared/config/externalUrls.ts

```diff
@@ -11,10 +11,13 @@ const externalUrls: Record<string, (id: string | number) => string> = {
   ENA: (id) => `//www.ebi.ac.uk/ena/data/view/${id}`,
   // protein centric
   InterPro: (id) => `https://www.ebi.ac.uk/interpro/protein/${id}`,
-  Pfam: (id) => `http://pfam.xfam.org/protein/${id}`,
+  Pfam: (id) => `https://www.ebi.ac.uk/interpro/protein/uniprot/${id}`,
```

Comment:
Thinking that maybe, at least for now, it should point to `https://www.ebi.ac.uk/interpro/protein/UniProt/${id}/entry/pfam/` maybe? Otherwise it might not be clear to people why they end up on the InterPro page for the protein if they don't immediately see Pfam data in the case it might be drowned within the other members databases.
What do you think?

Reply:
I agree, it is more of an InterPro landing. I will update it